stackslib/src/chainstate/stacks/db/transactions.rs

```diff
@@ -580,7 +601,7 @@ impl StacksChainState {
         origin_account: &StacksAccount,
         asset_map: &AssetMap,
         txid: Txid,
-    ) -> Result<bool, InterpreterError> {
+    ) -> Result<Option<String>, InterpreterError> {
```

Comment:
Can you update this function's rustdoc as well?

Reply:
Doh!

Reply:
c89cf7f112